python/tk_multi_workfiles/ui/entity_tree_form.py

```diff
@@ -19,6 +19,9 @@ def setupUi(self, EntityTreeForm):
         self.horizontalLayout = QtGui.QHBoxLayout()
         self.horizontalLayout.setContentsMargins(2, -1, 2, 1)
         self.horizontalLayout.setObjectName("horizontalLayout")
+        self.task_status_combo = QtGui.QComboBox(EntityTreeForm)
+        self.task_status_combo.setObjectName("task_status_combo")
+        self.horizontalLayout.addWidget(self.task_status_combo)
```

Comment:
What's it's default visibility? I'd vote for default hidden

Reply:
How can I achieve a default visibility in the UI form? Do you have an example?

Reply:
Not sure it will work but I think of this one: https://github.com/shotgunsoftware/tk-core/blob/v0.21.2/python/tank/authentication/login_dialog.py#L248

Reply:
The default state is visible.